legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/tds/tests/testTdsExtension.pure

```diff
@@ -481,4 +481,9 @@ function <<test.Test>>
 
    assertEquals('2014-12-01|356.0|3|TDSNull|TDSNull|false|false|false,2014-12-02|55.0|2|356.0|3|false|false|false,2014-12-03|71.0|2|55.0|2|false|true|false,2014-12-04|105.0|3|71.0|2|false|false|false,2014-12-05|5.0|1|105.0|3|false|false|false,2014-12-06|TDSNull|TDSNull|5.0|1|false|false|false',
       $relationalResult.rows->map(r|$r.values->makeString('|'))->makeString(',')); 
+}
+
+function <<test.Test>> meta::pure::tds::tests::extensions::testFirstNotNull():Boolean[1]
+{
+  assertEquals(1, [TDSNull, 1, 2]->meta::pure::tds::extensions::firstNotNull());
```

Comment:
do you think it makes sense to add a test also where we have all TDSNull?

Reply:
happt to add in follow up